.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": ["react", "es2015", "stage-0"],
-  "plugins": ["react-hot-loader/babel"]
+  "presets": ["react", "es2015", "stage-1"],
```

Comment:
dunno that's what google told me to do for getting decorators to work with babel 6